src/main/java/com/reckue/post/models/Rating.java

```diff
@@ -21,16 +22,16 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @Document
-public class Rating {
+public class Rating implements Serializable {
```

Comment:
There are definitely no other options for copy one object fields to another?

Reply:
I couldn't think of anything else